### ISSUE TITLE
Create a mechanism for thanking people electronically on behalf of the Foundation...

### ISSUE THREAD

**Issue by RobertLRead** [presidential-innovation-foundation/presidential-innovation-foundation.github.io#23]
I think we should create a "Thank You" page which is similar to the Donors page, but in contradistinction to it does not thank people for help to the Foundation per se, but is a way to thank people on behalf of individual PIFs.  I tried to get the White House to do this, and they were unresponsive, probably out of their very conservative nature in public communications.  What I imagine is a system by which a PIF, let's say Ben Balter version 3, whomever that is, sends us an email saying "Please thank Billy Bob Programmer for: open source code contributions to the Asteroid Challenge Project" or "Please thank Captain Smith for providing a room for the Coast Guard Workshop", etc.

Basically I think thanking people is the most important thing that we can do.
